alerts/alerts.go

```diff
@@ -266,7 +266,10 @@ func (a *originAlerter) RegisterAlert(ctx context.Context, alert Alert) error {
 	if alert.Data == nil {
 		alert.Data = make(map[string]any)
 	}
-	alert.Data["origin"] = a.origin
+	_, set := alert.Data["origin"]
```

Comment:
This doesn't seem quite right. Why would that be necessary unless you are wrapping an `originAlerter` in another `originAlerter` which should not happen.

Reply:
I didn't really see this as that much of an issue and it allowed dropping the double `alerts` vs `alertMgr` on the `bus`. The `bus.alerts` is not even used, but I didn't get rid of it either because we might then use the `alertMgr` on the `bus` at  one point and be without an origin. I reverted my changes so should be good.